src/components/Graph/Graph.tsx

```diff
@@ -128,15 +105,15 @@ const Graph: FC<GraphProps> = memo(
     }, []);
 
     // Expand all the current nodes by default if allowed
-    useEffect(() => {
-      if (evaluatedPathway.pathwayResults) {
-        for (const currentNode of evaluatedPathway.pathwayResults.currentStates) {
-          if (expandCurrentNode) {
-            if (currentNode) setExpanded(currentNode, true);
-          }
-        }
-      }
-    }, [expandCurrentNode, evaluatedPathway.pathwayResults, setExpanded]);
+    // useEffect(() => {
+    //   if (pathway.pathwayResults) {
+    //     for (const currentNode of pathway.pathwayResults.currentStates) {
+    //       if (expandCurrentNode) {
+    //         if (currentNode) setExpanded(currentNode, true);
+    //       }
+    //     }
+    //   }
+    // }, [expandCurrentNode, pathway.pathwayResults, setExpanded]);
```

Comment:
Is there a reason this was commented out and not deleted? Is the intention to eventually use this again once we define a current node of the builder?

Reply:
Yeah the idea was that we might want this piece again. But that's what source control is for isn't it. I'll delete it.